src/accordion/accordion.spec.ts

```diff
@@ -370,9 +370,9 @@ describe('ngb-accordion', () => {
     const contents = getPanelsContent(compiled);
     expect(contents.length).not.toBe(0);
 
-    contents.forEach((content: HTMLElement, idx: number) => {
+    contents.forEach((content, idx) => {
```

Comment:
Why did you remove the types here ? A little bit furthe, the same kind of code(i.e. : `contents.forEach((content: HTMLElement)`) has been left with the types. I think that we should keep the consistency in the whole file.

Reply:
thanks, will restore